### ISSUE TITLE
Date limit slider displays integers instead of text dates.

### ISSUE THREAD

**Issue by bmschmidt** [Bookworm-project/BookwormGUI#34]
E.g.: ![screen shot 2016-09-01 at 5 57 01 pm](https://cloud.githubusercontent.com/assets/1110758/18185854/8fadcf5a-706d-11e6-91fe-2ecef3410b38.png)

There was code to convert, for display, these wonky integer dates into some ISO-standard text display; not sure if it's still in there. It should be restored.

**Comment by organisciak** [comment]
What's the context for recreating this?

The (pre-merge to master) HT Bookworm at
https://bookworm.htrc.illinois.edu/develop/# seems to run it through
"fixTime()". Do you get any errors? Does setting a breakpoint on fixTime
actually stop the script? What browser?

**Comment by bmschmidt** [comment]
Sorry, should have linked.

The screenshot of the error I'm encountering is [this bookworm somewhere at Yale by @triplingual ; I'm uncertain of the exact commit being used used there](http://54.152.238.88/nyt12/#)

[Just tried to reproduce on the default Federalist test set, and things collapsed further--the x axis doesn't reproduce correctly.](http://benschmidt.org/federalist_bookworm/#)

The code on `fixTime()` looks like a custom patch at some point added purely for fields named `date_month`; but a field needing this particular fix might also be named something like `publish_month`, `publish_day_year`, etc. I already wrote some code to address this problem for the D3 bookworm [link to code block](https://github.com/bmschmidt/BookwormD3/blob/master/js/bookworm.js#L4116-L4140); maybe I should just port it over and overwrite the `fixTime()` function here?

But that's a separate issue. The key thing with Yale bookworm, I think, is that the conversion *is* happening, but only after the text is written to either side of the slider bar. It might be as simple as swapping two lines of code.

**Comment by pleonard212** [comment]
As a basic fix, this will do it:

`t = time_array[0]["dbfield"];`

instead of

`t = $("#time_measure").val();`

on this line: 

https://github.com/Bookworm-project/BookwormGUI/blob/master/app/bookworm.gui.js#L352

Caveat: the tooltips are still in integers as you drag the sliders.

Side note: month or even week-level granularity is one of the best features of Bookworm! :)


<img width="1186" alt="month" src="https://user-images.githubusercontent.com/1000161/201592455-e0538715-936a-408d-8034-39bbc6ee73cc.png">
